### DISCUSSION TITLE
Contiki-ng cc2538dk kit rpl does not work, There is no neighbor

### DISCUSSION THREAD

**Discussion by aceylan** [contiki-ng/contiki-ng#2448] [Q&A]
Hi all. 
I have cc2538dk and segger j-link jtag for flashing to cc2538dk. 
I have flashed rpl-border-router and coap servers to kits with segger j-flash successfully.
I have connected to border router with cable and "make TARGET=cc2538dk connect-router" command. You can see the picture attached. 
Other kits have coap server flashed are running near border-router. 
But border router says that there is no neighbor. 
You can see the pictures. 
Do you have any idea?
I have cc2538-cc2592 development kit. 

![Screenshot from 2023-05-22 12-19-25](https://github.com/contiki-ng/contiki-ng/assets/4823945/187b2fd8-e1d4-4d8c-bf63-dce42a10d857)
<img width="478" alt="cc2538dkandcomputer" src="https://github.com/contiki-ng/contiki-ng/assets/4823945/3f0b9f4d-95e1-4669-a2e3-09d0a90aeaf0">

**Comment by NestorESA** [comment]
I have similar problem, could you solve it?

**Comment by joakimeriksson** [comment]
Did you try any other examples first? To see that all do have the same configurations and have working communication?

**Reply by NestorESA** [reply]
Yes, I tried all the examples from platform folder, also tried the rpl-udp, these ones work but when I try any example that uses TSCH, they dont connect.
